index.d.ts

```diff
@@ -6,7 +6,7 @@ declare module 'react-intersection-observer' {
     children?: React.ReactNode | ((inView: boolean) => React.ReactNode)
 
     /** Render prop boolean indicating inView state */
-    render?(inView: boolean): React.ReactNode
+    render?: (inView: boolean) => React.ReactNode
```

Comment:
I see this syntax much more often than the one in the diff. I actually had no idea the previous one was valid, but apparently it is :grin: